composer.json

```diff
@@ -9,6 +9,7 @@
     "spryker/zend": "^2.1.0"
   },
   "suggest": {
+    "spryker/queue": "^1.2.0",
```

Comment:
Shouldnt those be in require-dev then as well?

Reply:
only if we make a test for it right?